samples/snippets/fsharp/azure/blob-storage.fsx

```diff
@@ -10,11 +10,6 @@ open System.Text
 //
 
 let storageConnString = "..." // fill this in from your storage account
-(*
-// To fetch the connection string from a configuration file.
-let storageConnString = 
-    CloudConfigurationManager.GetSetting("StorageConnectionString")
-*)
 
 //
 // Create some local dummy data.
```

Comment:
The comment on L88 refers to a different method than is being invoked.

Reply:
Are you looking at the "source" branch? I looked at dotnet:main and thought that as well, but the line refs worked fine (from what I saw) on the PR source branch

Reply:
I'm looking at the version in the PR diff.  _([ref](https://github.com/dotnet/docs/blob/792f40aafdaf0ec429532e04901410e2b74743d2/samples/snippets/fsharp/azure/blob-storage.fsx))_

![image](https://user-images.githubusercontent.com/913445/127505922-3300fe5c-4633-432c-a5c5-8da031cfd93d.png)

Line 88 mentions that `ListBlobsSegmentedAsync` is being called, but the call is to `GetBlobsByHierarchy`.    It doesn't appear that the error was introduced in this PR.

Reply:
In `main`, the comment is shifted down to L93, but the error is still present.

Reply:
oops. Sorry. Didn't even see that. I suppose it suggests that that comment (listing the method name) may not be required. Maybe better to say *what* the code is doing (or why) rather than a low-level look at what methods are being called etc.?

Reply:
Agreed; the comment needs to be revised, not just the method name updated.